lib/content-services/src/lib/viewer/components/alfresco-viewer.component.spec.ts

```diff
@@ -476,6 +476,20 @@ describe('AlfrescoViewerComponent', () => {
         //
     });
 
+    describe('originalMimeType', () => {
+
+        it('should set originalMimeType based on nodeData content', () => {
+            const nodeData = {
+                content: {
+                    mimeType: 'application/pdf'
+                }
+            };
+
+            component.originalMimeType = nodeData.content.mimeType;
+            expect(component.originalMimeType).toEqual(nodeData.content.mimeType);
```

Comment:
This unit test is setting one thing and then expecting the same thing which was set in previous test.

Reply:
I have modified the test cases